pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go

```diff
@@ -251,3 +294,17 @@ func (c *connectivityCheckController) enabled(operatorSpec *operatorv1.OperatorS
 		return false, nil
 	}
 }
+
+func setWithManagedByLabel(check *v1alpha1.PodNetworkConnectivityCheck) *v1alpha1.PodNetworkConnectivityCheck {
+	if check == nil {
+		return check
+	}
+	if check.Labels == nil {
+		check.Labels = make(map[string]string)
+	}
+	check.Labels[managedByLabelKey] = managedByLabelValue
+	check.ManagedFields = []metav1.ManagedFieldsEntry{{Manager: managedByLabelValue, Operation: metav1.ManagedFieldsOperationApply,
```

Comment:
I don't think you have to set the `ManagedFields` explicitly, it's managed by the apiserver. 
 https://kubernetes.io/docs/reference/using-api/server-side-apply/#field-management